src/v/config/configuration.cc

```diff
@@ -1411,6 +1411,13 @@ configuration::configuration()
       "Enables rack-aware replica assignment",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       false)
+  , storage_strict_data_init(
```

Comment:
Please mention the name of the property you're adding in the commit message

Reply:
Refactoring the commit to mention the name.

Reply:
I actually had it in originally. However, when doing
`
git commit -m "add `storage_strict_data_init` config paramter"
`
the shell tried to execute it lol.